scipy/ndimage/measurements.py

```diff
@@ -571,6 +571,18 @@ def _sum_centered(labels):
         return (counts, sums, sums_c)
 
 
+def sum_labels(input, labels=None, index=None):
+    """
+    Calculate the sum of the values of the array.
+
+    Notes
+    -----
+    This is an alias for `ndimage.sum`.  See the ``sum``
+    docstring for more details.
+
+    """
+    return sum(input, labels, index)
+
 def sum(input, labels=None, index=None):
```

Comment:
I think you want to rename this function to `sum_labels`, so it becomes the main implementation.

And then the "this is an alias" docstring can go onto a new `def sum` (as above)

Reply:
Done